bin/convertfont.toit

```diff
@@ -246,7 +246,7 @@ class FontReader:
 
   parse -> none:
     fd := file.Stream.for-read filename
-    reader := BufferedReader fd
+    reader := fd.in
```

Comment:
Should for-read just return the reader?

Or perhaps there should be a new function

```
reader := file.reader filename
```